sdk/eventhub/Azure.Messaging.EventHubs/src/Amqp/AmqpMessageConverter.cs

```diff
@@ -207,7 +207,7 @@ public virtual EventHubProperties CreateEventHubPropertiesFromResponse(AmqpMessa
 
         /// <summary>
         ///   Converts a given <see cref="AmqpMessage" /> response to a query for Event Hub partition properties into
-        ///   the corresponding <see cref="PartitionProperties" /> representation.
+        ///   the corresponding <see cref="PartitionProperties" /> representation;
```

Comment:
Same here. About the semicolon

Reply:
![](https://media0.giphy.com/media/J6IyI14oSTLr584B9a/giphy-downsized-medium.gif)

Where did they all come from?